<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPlotTest</class>
 <widget class="QWidget" name="CustomPlotTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox_left">
     <item>
      <property name="text">
       <string>ax+b</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>log</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sin</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_right">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>ax+b</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>log</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sin</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>自动刷新</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
